core/integration/odc/protos/odc.proto

```diff
@@ -1,236 +1 @@
-}
-
-// Run request
-message RunRequest {
-    string partitionid = 1; // Partition ID from ECS
-    uint64 runnr = 7; // Run number from ECS
-    uint32 timeout = 8; // Request timeout in sec. If not set or 0 than default is used.
-    // Either `topology`, `content` or `script` has to be set. If all or none is set then an error is returned.
-    string topology = 2; // Filepath to the XML DDS topology file
-    string content = 5; // Content of the XML DDS topology
-    string script = 6; // Shell commands to be executed by ODC in order to generate content of the XML DDS topology
-    string plugin = 3; // Name of the resource plugin registered in odc-server
-    string resources = 4; // Resource description 
-}
-
-// Update request
-message UpdateRequest {
-    string partitionid = 1; // Partition ID from ECS
-    uint64 runnr = 5; // Run number from ECS
-    uint32 timeout = 6; // Request timeout in sec. If not set or 0 than default is used.
-    // Either `topology`, `content` or `script` has to be set. If all or none is set then an error is returned.
-    string topology = 2; // Filepath to the XML DDS topology file
-    string content = 3; // Content of the XML DDS topology
-    string script = 4; // Shell commands to be executed by ODC in order to generate content of the XML DDS topology
-}
-
-// Shutdown request
-message ShutdownRequest {
-    string partitionid = 1; // Partition ID from ECS
-    uint64 runnr = 2; // Run number from ECS
-    uint32 timeout = 3; // Request timeout in sec. If not set or 0 than default is used.
-}
-
-// Key-Value property
-message Property {
-    string key = 1; // Property key
-    string value = 2; // Property value
-}
-
-// Set properties request
-message SetPropertiesRequest {
-    string partitionid = 1; // Partition ID from ECS
-    uint64 runnr = 4; // Run number from ECS
-    uint32 timeout = 5; // Request timeout in sec. If not set or 0 than default is used.
-    string path = 2; // Task path in the DDS topology. Can be a regular expression.
-    repeated Property properties = 3; // List of properties to be set
-}
-
-//
-// FairMQ device state change requests
-//
-
-// Configure request
-message ConfigureRequest {
-    StateRequest request = 1; // State change request. See StateRequest for details.
-}
-
-// Start request
-message StartRequest {
-    StateRequest request = 1; // State change request. See StateRequest for details.
-}
-
-// Stop request
-message StopRequest {
-    StateRequest request = 1; // State change request. See StateRequest for details.
-}
-
-// Reset request
-message ResetRequest {
-    StateRequest request = 1; // State change request. See StateRequest for details.
-}
-
-// Terminate request
-message TerminateRequest {
-    StateRequest request = 1; // State change request. See StateRequest for details.
-}
-
-//
-// Generic requests
-//
-
-// Status request
-message StatusRequest
-{
-    bool running = 1; // Select only running DDS sessions
-}
-
```

Comment:
Similar comment to the generated `pb.go` files, this one isn't generated but it's fetched from upstream. Either way no code changes should happen to this file, please exclude it from the PR. Let me know if you need help with this.